app/src/services/census/CensusStreamService.ts

```diff
@@ -54,10 +60,12 @@ export default class CensusStreamService implements ServiceInterface {
         });
 
         this.wsClient.on('reconnecting', () => {
+            this.messageTimer = null;
```

Comment:
```suggestion
            clearTimeout(this.messageTimer);
            delete this.messageTimer;
```

Maybe add a if statement?